### DISCUSSION TITLE
FR: Move between tabs by sliding content rows left/right?

### DISCUSSION THREAD

**Discussion by doron1** [kinghat/tabbed-card#120] [Feature Requests]
I'm not sure how easy/hard/impossible this is to implement, but it would have been cool if a slide left/right motion on the tabbed contents (rows) themselves would have moved between tabs. It feels natural and smooth, UX wise.
(Indeed those are separate cards, with their own controls, so this might be out there, but just thought I'd mention it.)

Thanks for this neat tool! Great enhancement to the UI, mainly on mobile.

**Comment by chintito4ever** [comment]
I second this feature.

**Comment by kinghat** [comment]
i could see this being an option in the future. there is also the venerable: https://github.com/bramkragten/swipe-card

**Reply by CoatsyJnr** [reply]
I would love this too
